custom/icds_reports/utils/__init__.py

```diff
@@ -431,7 +431,7 @@ def get_counseling(value):
     if counseling:
         return ', '.join(counseling)
     else:
-        return '--'
+        return DATA_NOT_ENTERED
```

Comment:
If they answer no to all these questions (as opposed to leaving them blank) will this show, if so, I do not believe this is correct